graylog2-server/src/main/java/org/graylog2/shared/rest/resources/system/RemoteDeflectorResource.java

```diff
@@ -20,6 +20,6 @@
 import retrofit2.http.POST;
 
 public interface RemoteDeflectorResource {
-    @POST("/system/deflector/cycle")
+    @POST("system/deflector/cycle")
```

Comment:
Could we potentially add a runtime check for the affected resources to log error messages?
Not sure where in the code this would have to go, the closest I could find is https://github.com/Graylog2/graylog2-server/blob/e638987fbf0e58e2211302a558a003fbfc18dff2/graylog2-server/src/main/java/org/graylog2/rest/RemoteInterfaceProvider.java#L41-L63